File: src/app/model/restaurant.ts
export class Restaurant{

    private id: number;
    private name: string;
    private capacityPerHour: number;
    private openHour: number;
    private closeHour: number;

    public constructor(){

        this.id = 0;
        this.name = "";
        this.capacityPerHour = 0;
        this.openHour = 0;
        this.closeHour = 0;
    }

    public getId(): number {
        return this.id;
    }
    public setId(value: number) {
        this.id = value;
    }

    public getName(): string {
        return this.name;
    }
    public setName(value: string) {
        this.name = value;
    }

    public getCapacityPerHour(): number {
        return this.capacityPerHour;
    }
    public setCapacityPerHour(value: number) {
        this.capacityPerHour = value;
    }

    public getOpenHour(): number {
        return this.openHour;
    }
    public setOpenHour(value: number) {
        this.openHour = value;
    }

    public getCloseHour(): number {
        return this.closeHour;
    }
    public setCloseHour(value: number) {
        this.closeHour = value;
    }
}

File: sb_tablereservation_rest/src/main/java/pti/sb_tablereservation_rest/model/Restaurant.java
package pti.sb_tablereservation_rest.model;

import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Column;
import org.springframework.data.relational.core.mapping.Table;

@Table(name = "restaurants")
public class Restaurant {

	@Id
	@Column(value = "id")
	private Integer id;
	
	@Column(value = "name")
	private String name;
	
	@Column(value = "capacity_per_hour")
	private int capacityPerHour;
	
	@Column(value = "open_hour")
	private int openHour;
	
	@Column(value = "close_hour")
	private int closeHour;

	
	
	public Restaurant(Integer id, String name, int capacityPerHour, int openHour, int closeHour) {
		super();
		this.id = id;
		this.name = name;
		this.capacityPerHour = capacityPerHour;
		this.openHour = openHour;
		this.closeHour = closeHour;
	}
	

	public Integer getId() {
		return id;
	}

	public void setId(Integer id) {
		this.id = id;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public int getCapacityPerHour() {
		return capacityPerHour;
	}

	public void setCapacityPerHour(int capacityPerHour) {
		this.capacityPerHour = capacityPerHour;
	}

	public int getOpenHour() {
		return openHour;
	}

	public void setOpenHour(int openHour) {
		this.openHour = openHour;
	}

	public int getCloseHour() {
		return closeHour;
	}

	public void setCloseHour(int closeHour) {
		this.closeHour = closeHour;
	}
	
	
}
